packages/api/cms-api/src/user-permissions/user.resolver.ts

```diff
@@ -9,6 +9,12 @@ import { UserPermissionsService } from "./user-permissions.service";
 @ObjectType()
 class PaginatedUserList extends PaginatedResponseFactory.create(User) {}
 
+@ObjectType()
+class UserPermissionsOptions {
+    @Field()
+    showAllUsersButton: boolean;
+}
```

Comment:
As of now, we don't have this kind of options in the GraphQL schema, but somewhat duplicate it in the Admin and API. For instance, this option could also be solved via a prop in the Admin. Should we reconsider our existing approach?